### ISSUE TITLE
ModuleNotFoundError: No module named 'PIL' (totally new to python)

### ISSUE THREAD

**Issue by JacksonRFilms** [carykh/jumpcutter#83]
I installed Python 2.5.3, 3.4.1 and the newest Python, and ffmpeg is working, and pip seems to work. When I run `pip list` it shows `Pillow (2.5.3)` exists. I have the jumpcutter-master folder in my downloads folder (does it have to be in a specific location?) I drag jumpcutter.py into cmd and run it, and I get this:
```
C:\Users\pitot>C:\Users\pitot\Downloads\jumpcutter-master\jumpcutter-master\jumpcutter.py
Traceback (most recent call last):
  File "C:\Users\pitot\Downloads\jumpcutter-master\jumpcutter-master\jumpcutter.py", line 2, in <module>
    from PIL import Image
ModuleNotFoundError: No module named 'PIL'
```
I'm totally new to Python and the only coding experience I have is one semester of Java, so I barely understand anything I've read on the internet about this issue. Can you guys dumb it down a little for me? Do I have to add more things to the "PATH" variable or something? Do I need a special python terminal or is cmd prompt ok?

**Comment by pniedzwiedzinski** [comment]
Check if `pip --version` and `python --version` are showing the same path

**Comment by TemporarilyOffline** [comment]
I got this error also, but I ran pip install <requirement found in requirements.txt> and it works fine now.

Make sure to use the version of pip that goes with the version of python you have installed (I have several versions installed)

**Comment by luizfdflores** [comment]
Make sure to install Pillow at the right version `pip install Pillow@2.5.3`

**Comment by elff1493** [comment]
I don’t think pil is even used, you can probably delete that line

**Comment by Hellzbellz123** [comment]
this application doesnt use python 2 does it?

**Comment by VR3Tricarico** [comment]
Where do you download Pillow?

**Comment by Hellzbellz123** [comment]
pip install pil

**Comment by Hellzbellz123** [comment]
https://www.youtube.com/watch?v=a0oa3Qiz5tw

**Comment by GamerLlama** [comment]
You have to do "pip install pillow" instead of "pip install PIL"

**Comment by ozanhalisilter** [comment]
Have you tried to install pillow under that file?
